pkg/util/mount/mount.go

```diff
@@ -143,6 +142,23 @@ func GetMountRefs(mounter Interface, mountPath string) ([]string, error) {
 	return refs, nil
 }
 
+// GetMountRefsByDev finds all references to the device provided
+// by devPath; returns a list of paths.
+func GetMountRefsByDev(mounter Interface, devPath string) ([]string, error) {
+	mps, err := mounter.List()
+	if err != nil {
+		return nil, err
+	}
+	// Find all references to the device.
+	var refs []string
+	for i := range mps {
+		if mps[i].Path == devPath || mps[i].Device == devPath {
```

Comment:
@msau42 Exactly, but I need to use /mnt/disks/vol1 as a key in my search. So all references to /mnt/disks/vol1 found in all pods mounts are counted.